code/modules/mob/living/basic/lavaland/raptor/_raptor.dm

```diff
@@ -19,16 +19,22 @@ GLOBAL_LIST_EMPTY(raptor_population)
 
 #define HAPPINESS_BOOST_DAMPENER 0.3
 
-/mob/living/basic/mining/raptor
+/mob/living/basic/raptor
 	name = "raptor"
 	desc = "A trusty, powerful steed. Taming it might prove difficult..."
 	icon = 'icons/mob/simple/lavaland/raptor_big.dmi'
 	speed = 2
 	mob_biotypes = MOB_ORGANIC|MOB_BEAST
-	maxHealth = 400
-	health = 400
+	maxHealth = 250
+	health = 250
 	melee_damage_lower = 10
 	melee_damage_upper = 15
+	combat_mode = TRUE
+	status_flags = NONE
+	mob_size = MOB_SIZE_LARGE
+	unsuitable_atmos_damage = 0
```

Comment:
I'm fine with this remaining, but worth checking... Is there a reason for them to be spaceproof?
Can just set them to be able to survive in Lavaland/Icebox but not with 0 pressure or temperature.

Reply:
good idea

Reply:
actually i dont know if theres a way to make this work off pressure